plugins/modules/vmware_guest_cross_vc_clone.py

```diff
@@ -307,6 +312,9 @@ def populate_specs(self):
         creds.password = self.destination_vcenter_password
         self.service_locator.credential = creds
 
+        if self.destination_vcenter_thumbprint:
+            self.service_locator.sslThumbprint = self.destination_vcenter_thumbprint
```

Comment:
```suggestion
        if self.destination_vcenter_validate_certs is False:
            self.service_locator.sslThumbprint = self.get_cert_fingerprint(self.destination_vcenter, self.destination_vcenter_port,
                                                                           self.module.params['proxy_host'], self.module.params['proxy_port'])
```

@ironbishop I think this will fix the issue without a new parameter. At least, it worked for me.